src/gluonts/itertools.py

```diff
@@ -332,3 +332,10 @@ def trim_nans(xs, trim="fb"):
                 break
 
     return xs[start:end]
+
+
+def inverse(dct):
```

Comment:
Can we add the data types here? 

`def inverse(dct: Dict) -> Dict:`

Reply:
Good point, added.